radicle-node/src/service/features.rs

```diff
@@ -0,0 +1,134 @@
+use std::{fmt, ops};
+
```

Comment:
Is this included in its own module because of file size?  The module name spacing may be better missing one level.

For instance service::Features::None vs. service::features::NodeFeatures::None

Reply:
Or `service::features::Node::None`, if there is going to be another service feature type -- leaving room for that.

Reply:
Yeah, agree this isn't optimal. Here's some other alternatives:

1. since these are constants, have a `constants` module and then `radicle_node::constants::NodeFeatures::SEED`
2. have them at the top level: `radicle_node::Features::SEED`
3. have them in the `node` module of `radicle`: `radicle::node::Features`
4. bitcoin calls these `ServiceFlags`, perhaps it makes sense in this context: `radicle_node::service::Flags`

Reply:
My personal preference would be 3. I love a good qualified hierarchy of modules :)